draft-pda-protocol.md

```diff
@@ -1173,11 +1172,16 @@ mitigations available to aggregators also apply to the leader.
    aggregations, joint randomness, arithmetic circuits).
 1. Collectors may trivially defeat availability by discarding output shares
    submitted by aggregators.
+1. Known input injection. Collectors may collude with clients to send known
+   input to the aggregators, allowing collectors to shrink the effective
+   anonymity set by subtracting the known inputs from the final output.
```

Comment:
I would work in the term "Sybil attack" here. While we're at it, let's add a reference for Sybil attacks (i.e., `{{sybil}}`). I think the canonical reference is https://link.springer.com/chapter/10.1007/3-540-45748-8_24.

Reply:
Done